src/pages/projectSettings/CreateProjectModal.tsx

```diff
@@ -29,7 +29,9 @@ export const CreateProjectModal: React.VFC<Props> = ({
   owner,
   repo,
 }) => {
-  const dispatchToast = useToastContext();
+  // const dispatchToast = useToastContext();
+  const { error: errorToast, success, warning } = useToastContext();
```

Comment:
we haven't been using this destructuring pattern for things like toasts. can we use the previous pattern?

Reply:
done